src/helm/benchmark/window_services/window_service_factory.py

```diff
@@ -52,10 +52,14 @@ def get_window_service(model_name: str, service: TokenizerService) -> WindowServ
         engine: str = model.engine
 
         window_service: WindowService
+        # Catch any HuggingFace models registered via the command line flags
         huggingface_model_config = get_huggingface_model_config(model_name)
+        # Catch a select list of local HuggingFace models not registered upfront (only the ones defined in models.py)
+        if not huggingface_model_config and organization == "local":
+            huggingface_model_config = HuggingFaceModelConfig.from_path(f"{LOCAL_MODEL_DIR}/{engine}")
```

Comment:
Don't think we should special case `local`. See my comment later about module initialization-time model registration in `huggingface_model_registry`.